.github/workflows/ci.yml

```diff
@@ -46,13 +46,31 @@ jobs:
           # latest dev build
           # https://www.sublimetext.com/dev
           - sublime-channel: dev
-            sublime-build: 4134
+            sublime-build: 4137
             optional: false
 
     steps:
 
+      - name: Git config (global)
+        run: |
+          git config --global user.name 'github-actions[bot]'
+          git config --global user.email '41898282+github-actions[bot]@users.noreply.github.com'
+          git config --global init.defaultBranch master
```

Comment:
Is that needed for `actions/checkout@v3`?

Reply:
The primary task of actions like `actions/checkout` is to avoid us needing explicitly run shell scripts. If we'd want to be explicit we could just do a git pull via bash manually.

So I'd expect `actions/checkout` to configure whatever is needed to pull required assets for testing.